scripts/check-package-dependencies.js

```diff
@@ -0,0 +1,131 @@
+const fs = require('fs');
```

Comment:
I realize we probably want this to be a low-dependency script, but maybe it is still worth writing it in typescript anyway